src/main/scala/coreplex/Coreplex.scala

```diff
@@ -65,21 +80,82 @@ abstract class Coreplex(implicit val p: Parameters, implicit val c: CoreplexConf
 }
 
 class DefaultCoreplex(tp: Parameters, tc: CoreplexConfig) extends Coreplex()(tp, tc) {
+  class TileWrapper(tileBuilder: (Bool, Parameters) => Tile, _clock: Clock, _reset: Bool)
+      (implicit p: Parameters) extends Module(Some(_clock), Some(_reset)) {
+
+    val wrappedTile = tileBuilder(reset, p)
+    val io = new Bundle {
+      val tile = wrappedTile.io.cloneType
+      val ext_clock = Clock(INPUT)
+      val ext_reset = Bool(INPUT)
+    }
+
+    io.tile.uncached <> wrappedTile.io.uncached.map(AsyncClientUncachedTileLinkTo(io.ext_clock, io.ext_reset, _))
+    io.tile.cached <> wrappedTile.io.cached.map(AsyncClientTileLinkTo(io.ext_clock, io.ext_reset, _))
+    wrappedTile.io.interrupts := LevelFrom(io.ext_clock, io.tile.interrupts)
```

Comment:
Ah, I see, these are synchronizing between PLIC and the Tile. Ok, then the case we need to worry about is that when you claim the interrupt, the PLIC/TL doesn't return from the 'complete write before the interrupt that you see in the Tile post-synchronizer is cleared. *Probably* this is true, but you need to convince yourself and put appropriate requires (like the synchronizer depths should match, have safety margin, or whatever).